toast.yml

```
@@ -1,12 +1,35 @@
-image: gcr.io/windmill-public-containers/tilt-toast@sha256:232b4f5b187da1bedc3bd6fcc0d07a8d33c169fa9430ac0105da7efb5a65ca9f
+image: gcr.io/windmill-public-containers/tilt-toast@sha256:f3dc747f3861ae274b72943ea511a46e9e760928507191fb8179480ae804933d
 tasks:
   proto:
     input_paths:
       - pkg/webview/view.proto
     output_paths:
       - pkg/webview/view.pb.go
       - pkg/webview/view.pb.gw.go
-    command: protoc -I. -I/usr/include -I"$(go env GOPATH)"/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --grpc-gateway_out=logtostderr=true:. --go_out=plugins=grpc,paths=source_relative:. pkg/webview/view.proto && goimports -local github.com/windmilleng/tft -w pkg/webview/view.pb.go
+      - pkg/webview/view.swagger.json
+    command: |
+      set -euo pipefail
+      mkdir -p web/src
+      protoc \
+       -I. \
+       -I/usr/include \
+       -I"$(go env GOPATH)"/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
+       --grpc-gateway_out=logtostderr=true:. \
+       --swagger_out=logtostderr=true:. \
+       --go_out=plugins=grpc,paths=source_relative:. \
+       pkg/webview/view.proto
+      goimports -local github.com/windmilleng/tilt -w pkg/webview/view.pb.go
+      
+  proto-ts:
+    dependencies:
+      - proto
+    input_paths:
+      - pkg/webview/view.swagger.json
+    output_paths:
+      - web/src/view.d.ts
+    command: |
+      npx @manifoldco/swagger-to-ts pkg/webview/view.swagger.json --camelcase --output web/src/view.d.ts
```

If we want to continue with using yarn everywhere I think we can change this to use [yarn dlx](https://next.yarnpkg.com/cli/dlx):

```suggestion
      yarn dlx @manifoldco/swagger-to-ts pkg/webview/view.swagger.json --camelcase --output web/src/view.d.ts
```

---

done